packages/core/src/sass/globals.scss

```diff
@@ -47,7 +47,7 @@ $uikit-maxwidth: 42em; //Answer to life, the universe, and everything (keeps thi
 $uikit-colour-Hero: #115361 !global; //color variables (yes! the actual spelling doesn't have a u)
```

Comment:
**I'm concerned that you guys aren't using Sass partials!** All Sass files should be prefixed with an `_` like `_globals.scss` otherwise you'll recreate Sass ecosystem. Need to fix.